src/test/java/carRacing/domain/CarTest.java

```diff
@@ -0,0 +1,39 @@
+package carRacing.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void setUp(){
+        this.car = new Car();
+    }
+
+    @Test
+    @DisplayName("자동차 인스턴스를 생성할 수 있다")
+    public void constructorCreatesNewCar(){
+        assertEquals(car.getScore(),0);
+        Car updatedCar = new Car(car);
+        assertEquals(updatedCar.getScore(), 0);
```

Comment:
```suggestion
        assertThat(updatedCar).isEqualTo(new Car(0));
```
getter 를 사용하지 않고 상태비교를 하려면 어떤 방법이 있을까요? 

추가로 junit 의 [assertEquals](http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(long,%20long)) 은 앞의 인자가 예상값, 뒤의 인자가 실제값입니다. 저도 자주 헷갈려요 ☹️ 그래서 이런부분이 좀 더 명확한 assertJ 를 사용합니다. 과정에서도 assertJ 를 메인 테스팅 라이브러리로 사용하고 있으니 활용해 보시는게 어떨까요?